bin/stronghold-cli

```diff
@@ -121,6 +121,7 @@ opts = GetoptLong.new(
   [ '--next', '-n', GetoptLong::OPTIONAL_ARGUMENT ],
   [ '--set', '-s', GetoptLong::REQUIRED_ARGUMENT ],
   [ '--set-json', '-j', GetoptLong::OPTIONAL_ARGUMENT ],
+  [ '--replace-json', '-R', GetoptLong::OPTIONAL_ARGUMENT ],
```

Comment:
What was your motivation to make this an upper case `R`? The current convention looks like upper case is only used if the lower case is already taken, e.g. `J` would have been reasonable here. But I think `r` is fine too.

Reply:
I originally called it just `--replace` and `-r` is taken by the `--run` option so I made it `-R`. I agree that `-J` makes more sense now that it's called `--replace-json` so I'll update that.

Reply:
Ah sorry, I had missed the `-r` for `--run`. Yeah I think `J` is the best option then. `-R` makes it seem like it might be related to `--run`.